docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -21,7 +21,13 @@ To use this guide, you must first [create an Azure storage account](/azure/stora
 
 The samples in this article can be used in either an F# application or an F# script. To create an F# script, create a file with the `.fsx` extension, for example `blobs.fsx`, in your F# development environment.
```

Comment:
Does the `#` on L20 need to be encoded?  It was on L8, is there a difference in the rendering?

Reply:
nit:  The Microsoft Style Guidelines specify that headings should use "sentence-style" capitalization, which those on L1-20 do not.   _([ref](https://docs.microsoft.com/style-guide/scannable-content/headings#formatting-headings))_

Reply:
Yes, `#` on L20 need to be encoded, they have no difference in the rendering, except `#` on L8 was at the end of the sentence. For consistency, I suggest adding `\`.